web/src/components/ErrorBoundary.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const OopsMessage = () => (
+  <div className="site-body ds-l-container">
+    <main id="start-main-content">
+      <div className="ds-u-padding-top--2">
+        <h1 data-testid="errorboundary">
+          Oops! Looks like something went wrong.
+        </h1>
+        <p>You might want to double-check your link and try again. (404)</p>
+      </div>
+    </main>
+  </div>
+);
+
+class ErrorBoundary extends Component {
```

Comment:
Does this have to be a class component?

Reply:
When I was looking at the documentation that's what it looked like. I can try to remove it and see what happens? But it mad sense since it wraps around a component.

Reply:
Looked into it, and React doesn't have a hook `getDerivedStateFromError` so it isn't callable in a functional style component.